core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -67,7 +67,7 @@
               <a class="dropdown-item nav-link oppia-navbar-tab"
                  style = "color: #00376d"
                  ng-href="<[url]>">
-                 {{tabName}}
+                 <[tabName]>
```

Comment:
@vojtechjelinek, do we still use `{{}}` anywhere in the codebase? If not then I think we can add a lint test for this.

Reply:
We use it in some demonstration file in the logic interaction, but this file can be added as an exception.